pkg/backingstore/backingstore.go

```diff
@@ -530,6 +545,66 @@ func RunDelete(cmd *cobra.Command, args []string) {
 	}
 }
 
+// RunRemovePendingPods runs a CLI command
+func RunRemovePendingPods(cmd *cobra.Command, args []string) {
+	log := util.Logger()
+
+	if len(args) != 1 || args[0] == "" {
+		log.Fatalf(`❌ Missing expected arguments: <backing-store-name> %s`, cmd.UsageString())
+	}
+
+	o := util.KubeObject(bundle.File_deploy_crds_noobaa_io_v1alpha1_backingstore_cr_yaml)
+	backStore := o.(*nbv1.BackingStore)
```

Comment:
how will the backingstore reconciler will behave once you are messing with its pods? also, don't you need to update the numVolumes?

Reply:
changing the numVolumes won't help as the reconcile will start running during the deletion of the pods and try to recreate them. So we need for him to first change the numVolumes wait for it to stable. and then run this command